contracts/extbridge/ext_bridge.sol

```diff
@@ -82,7 +82,7 @@ contract ExtBridge is BridgeTransferERC20, BridgeTransferERC721 {
     )
         public
     {
-        if (_extraData.length > 0) {
+        if (_extraData.length == 32) { // size check of uint256
```

Comment:
if this condition fails with (length > 0 && length != 32), how to about reverting this method?

Reply:
@aidan-kwon What do you think? :)
In this example, if we want not to allow other extraData (not uint256), original code looks better.